src/plugins/orangehrmCorePlugin/Api/V2/Request.php

```diff
@@ -50,7 +50,7 @@ class Request
      * Query string parameters ($_GET)
      * @var ParameterBag
      */
-    protected ParameterBag $query;
+    public ParameterBag $query;
```

Comment:
Oops, why making this public, use getter method

Reply:
Actually when working, phpstorm suggested that, forgot to revert back. Done